### DISCUSSION TITLE
ucryptolib's desktop "twin"?

### DISCUSSION THREAD

**Discussion by moschles** [micropython/micropython#9538] [Libraries & Drivers]
Does anyone happen to know which desktop library `ucryptolib` was loosely based on?   I need to decrypt a payload from an embedded device, and I would like to keep the libraries as compatible as possible between my device and my server.  Thanks.

**Comment by jimmo** [comment]
Good question... Other than the creation of the `Cipher` object, the actual use of the objects via the encrypt/decrypt methods match PyCryptoDome.
